File: 1894-merge-strings-alternately/merge-strings-alternately.py
class Solution(object):
    def mergeAlternately(self, word1, word2):
        """
        :type word1: str
        :type word2: str
        :rtype: str
        """
        l1=len(word1)
        l2=len(word2)
        bigg_word = word1 if len(word1) > len(word2) else word2
        res=""
        for i in range(min(l1,l2)):
            res+=word1[i]+word2[i]
        return res+bigg_word[min(l1,l2):]

File: 1894-merge-strings-alternately/merge-strings-alternately.java
class Solution {
    public String mergeAlternately(String word1, String word2) {
        StringBuilder merged=new StringBuilder();
        String bigWord=new String();
        int length_1=word1.length();
        int length_2=word2.length();
        int min_length=Math.min(length_1,length_2);
        if(length_1>length_2){
            bigWord=word1;
        }else{
            bigWord=word2;
        }
        for(int i=0;i<min_length;i++){
            merged.append(word1.charAt(i)).append(word2.charAt(i));
        }
        for(int j=min_length;j<bigWord.length();j++){
            merged.append(bigWord.charAt(j));
        }
        return merged.toString();
        
    }
}